modules/nf-core/krakenuniq/preloadedkrakenuniq/meta.yml

```diff
@@ -25,6 +25,11 @@ input:
         type: file
         description: List of input files containing sequences. All of them must be either
           in FASTA or FASTQ format.
+    - prefixes:
+        type: string
+        description: >
+          List of sample identifiers or filename prefixes. Must correspond in order and
```

Comment:
Maybe give a suggestio how to fix the order (maybe with `.sort`? Although I can imagine in some very rare cases someone might be weird and not name things such as pairs properly)

Reply:
I think that in the normal case, prefixes will be constructed from samples as they come out of a channel and thus they will be in the same order. I do not want to suggest something here that potentially makes the use more complicated than necessary and potentially leads to user errors.

I am in favor of creating a batching function and having that properly tested, but I don't know where we would provide that from?